upgrade/1.2.1/upgrade_network.md

```diff
@@ -0,0 +1,102 @@
+# Upgrade and validate Switch Configurations
+
+## Prerequisites
+
+* SSH access to the switches or the running configuration file
+* Upgrade CANU: [Install/Upgrade CANU](../../operations/network/management_network/canu_install_update.md)
```

Comment:
Perhaps "Upgraded" or "install latest"?

Reply:
Upgrade or install latest version of CANU